Product bulk actions

NTWEXO opened the pull request:
### 1. Why is this change necessary?
Add more bulk action options

### 2. What does this change do, exactly?
Add the bulk option to activate or deactivate products, the already excisting bulk option to delete products have been added under the dropdown

### 3. Describe each step to reproduce the issue or behaviour.
Open product listing page in the administration panel.
select all products you want to affect (use the checkmark to the left)
a dropdown will appear i the top (where it used to say "delete")
select you desired action

### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

hlohaus commented:
PS: Da der Router noch für das alte Plugin-System geschrieben wurde, gibt es dafür Setter und Getter im Router. Man kann es also schon so erweitern:
https://forum.shopware.com/discussion/comment/138402/#Comment_138402
Trotzdem wäre die andere Lösung netter. :+1:

Haberkamp committed:
NEXT-8189 - fix after order link in mail templates

Commit:
next-6981 - add payment error handling to checkout

Commit:
next-6981 - add payment error handling to checkout

NTWEXO committed:
added bulk actions for activate and deactivate products and started on changing categoreis

pweyck committed:
NTR - Workaround dev branch selection by setting it in the repo

shyim committed:
NEXT-8551 - Allow only price definitions with permission

pweyck committed:
NTR - Checkout current branch in travis

shyim committed:
NEXT-8551 - Allow only price definitions with permission

pweyck committed:
Merge branch 'next-8551/allow-only-pricedefinitions-with-permission' into '6.2'

NEXT-8551 - Allow only price definitions with permission

See merge request shopware/6/product/platform!2232

pweyck committed:
Merge branch 'next-6981/payment-error-handling-6-2' into '6.2'

next-6981 - add payment error handling to checkout

See merge request shopware/6/product/platform!2237

shyim committed:
NTR - Github actions

pweyck committed:
Revert "next-6981 - add payment error handling to checkout"

This reverts commit ec8590c37dbc21b40ae2bdc9e15e21e07cf4b0f6.

Commit:
Merge branch 'next-8189/fix-after-order-link-in-mail-templates' into '6.2'

NEXT-8189 - fix after order link in mail templates

See merge request shopware/6/product/platform!2148

shyim committed:
NTR - Run tests in multiple test suites

shyim committed:
NTR - Add Jest

OliverSkroblin committed:
NEXT-8571 - Remove http client dependencies

OliverSkroblin committed:
NEXT-8571 - Added new StoreApiClient which allows to send requests with context token and access key

pweyck committed:
Merge branch 'next-698/add-payment-error-handling-2' into '6.2'

next-6981 - add payment error handling to checkout

See merge request shopware/6/product/platform!2251

pweyck committed:
Merge branch 'next-8571/optimize-http-client' into '6.2'

NEXT-8571 - Remove http client dependencies

See merge request shopware/6/product/platform!2242

Ion-manden committed:
Merge remote-tracking branch 'origin/6.2-BoostDay' into product-bulk-actions

Ion-manden committed:
Revert "Merge remote-tracking branch 'origin/6.2-BoostDay' into product-bulk-actions"

This reverts commit 17ac5aa7a3fbc09c241cac4661618e60ba192d3f.

tinect committed:
NEXT-7780 Add titles to editor toolbar, fixes #669

t2oh4e committed:
NEXT-8428 - Add link to customer on order overview in administration. Fixes #802

Commit:
NEXT-8594 - Add possibility to delete orders on sw-order-list - Fixes #872

SeboBlock committed:
NEXT-7442 fixes #851 add guest pill to customer list and detail view next to the customer name

Commit:
NEXT-8437 - correct product stream demo data generation - Fixes #672

wexomep committed:
NEXT-7417 - Fix replace comma automatically by dot in price fields - Fixes #823

hit-lember committed:
NEXT-7788 - Add config for disabling reviews, fixes #852

Commit:
NEXT-8435 Fix duplicate script entrypoints

#486

Commit:
NEXT-8285 - fix JS Error in Admin Area Users & Rights (New User) - Fixes #820

JoshuaBehrens committed:
NEXT-8603 Don't load multiple bundles

Fixes #858

tbkruse committed:
NEXT-8260 Fix Sorting the properties in the filter has wrong order #827

benjamin-ott committed:
NEXT-8616 - adds image lazy loading to zoom modal - fixes https://github.com/shopware/platform/issues/885

Commit:
NEXT-7687 Fixes #832 Fix invoice numbers in credit memo modal

mitelg committed:
Merge pull request #892 from jochenmanz/feature/NEXT-8437

NEXT-8437 - correct demo data product stream filters

mitelg committed:
Merge pull request #894 from wexo/NEXT-7471_Replace-comma-auto

NEXT-7417 - Fix replace comma automatically by dot in price fields

Commit:
NEXT-8438 Fixes #769 add event listener to close flyout when cursor leaves window, also repair flyout flickering

xPand4B committed:
Merge pull request #893 from hhoechtl/fix/NEXT-8435

NEXT-8435 Test for duplicate script entrypoints

BrentRobert committed:
NEXT-7087 Fixed type error
Fixes #817

BrentRobert committed:
NEXT-8493 Added missing blocks and plugins
Fixes #855

alexbaat committed:
NEXT-8597 - Translate scale units in admin Fixes #869

mitelg committed:
Merge pull request #889 from t2oh4e/feature/NEXT-8428-add-customer-link

NEXT-8428: Add customer link to customer overview

pantrtxp committed:
Merge pull request #895 from runelaenen/NEXT-8285

NEXT-8285 - JS Error in Admin Area Users & Rights (New User)
Fixes #820

jleifeld committed:
Merge pull request #901 from runelaenen/NEXT-7687

NEXT-7687 Fixes #832

pantrtxp committed:
Merge pull request #904 from StephanAltmann85/NEXT-8594/issue-872

NEXT-8594 - Add possibility to delete orders on sw-order-list

alexbaat committed:
NEXT-8340 - Fix the media file name direction in media library Fixes #810

shopwareBot committed:
Merge pull request #900 from BrentRobert/NEXT-7087

NEXT-7087 #817 Fixed type error in SalesChannelCmsPageRepository sort…

shopwareBot committed:
Merge pull request #907 from BrentRobert/NEXT-8493

NEXT-8493 Added missing blocks and JS plugins

vancaw1 committed:
NEXT-8195 Fix timezone of order time in ordergrid

shopwareBot committed:
Merge pull request #920 from wannevancamp/feature/NEXT-8195

NEXT-8195 Fix timezone of order time in ordergrid

wexollm committed:
NEXT-6407 add @media-upload-sidebar-open to sw-manufacturer-detail page, fixes #815

FalkoHilbert committed:
NEXT-5263 - add posibility to configure accepted file types in media-upload component
and configure media-page component to default allow all filetypes
Fixes #806

t2oh4e committed:
NEXT-8436 - Remove alpha numerical filter from quick add. Fixes #680

shopwareBot committed:
Merge pull request #912 from FalkoHilbert/NEXT-5263-pdf-upload

NEXT-5263 - PDF Upload

SRaromicon committed:
NEXT-8596 Adding Ability to hide/show Properties in Product Filter in Product Listing.

shyim committed:
Merge pull request #913 from pixolith/feature/NEXT-7442-display-of-accountless-customers-in-admin

NEXT-7442 fixes #851 add guest pill to customer list and detail view …

shyim committed:
Merge pull request #928 from t2oh4e/feature/NEXT-8436

NEXT-8436 - Remove alpha numerical filter from quick add. Fixes #680

jkrzefski committed:
NEXT-8546 - Remove faulty early return from SystemConfigService::getSubArray (#925)

Fixes #861

mitelg committed:
Merge pull request #918 from alexbaat/bugfix/NEXT-8340

NEXT-8340 - Fix the media file name direction in media library Fixes …

mitelg committed:
Merge pull request #903 from pixolith/NEXT-8438

NEXT-8438: Fixes #769 | add event listener to close flyout when cursor leaves window, also repair flyout flickering

kiplingi committed:
NEXT-8288 - Newsletter GDPR, fixes #819

jleifeld committed:
Merge branch '6.2-BoostDay' into feature/NEXT-8597

jleifeld committed:
Merge branch '6.2-BoostDay' into feature-669

shopwareBot committed:
Merge pull request #936 from kiplingi/NEXT-8288

NEXT-8288 Newsletter GDPR

jleifeld committed:
Merge pull request #686 from tinect/feature-669

[NEXT-7780] add titles to text-editor-toolbar

jleifeld committed:
Merge branch '6.2-BoostDay' into feature/NEXT-8597

jleifeld committed:
Merge pull request #910 from alexbaat/feature/NEXT-8597

NEXT-8597 Translate scale units in admin

shopwareBot committed:
Merge pull request #919 from sebi007/NEXT-7788

NEXT-7788 - Add config for disabling reviews

ca-tb committed:
NEXT-8598 - Add plural to packaging unit - Fixes #935

Commit:
Merge pull request #902 from benjamin-ott/next-8616/zoom-modal-lazy-load

NEXT-8616 - fixes https://github.com/shopware/platform/issues/885

OliverSkroblin committed:
Merge pull request #930 from SRaromicon/feature/NEXT-8596

NEXT-8596 Adding Ability to hide/show Properties in Product Filter in Product Listing. Fixes #870

OliverSkroblin committed:
Merge pull request #908 from tbkruse/bugfix/NEXT-8260

NEXT-8260 Fix Sorting the properties in the filter has wrong order #827

pweyck committed:
Merge pull request #896 from JoshuaBehrens/bugfix/ignore-duplicate-bundle-loading

NEXT-8603 Don't load multiple bundles from plugins

wexotk committed:
NEXT-8592 - Fixes #879 change customer password from administration (#922)

Commit:
Merge pull request #926 from wexo/NEXT-6407_upload_media_on_manufacturer_detail

NEXT-6407 - Upload media on manufacturer detail

Ion-manden committed:
cs fix

fix double quotes and debuging code

fix indentations

remove assign/remove cateogry bulk option

Ion-manden committed:
Revert "Merge branch '6.2-BoostDay' into product-bulk-actions"

This reverts commit b38dcb8b7b0447837813b969c50c1ccbbc9558e3.

Ion-manden committed:
Merge branch '6.2-BoostDay' into product-bulk-actions
